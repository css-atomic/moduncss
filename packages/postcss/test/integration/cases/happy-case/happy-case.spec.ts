import { expect, test } from 'vitest'
import { processFile } from '../../helper/process'

test('happy case', () => {
  expect(processFile('happy-case/data/1.css')).resolves.toMatchSnapshot('happy case')
})
