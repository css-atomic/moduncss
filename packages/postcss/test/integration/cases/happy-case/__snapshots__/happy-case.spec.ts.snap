// Vitest Snapshot v1

exports[`happy case > happy case 1`] = `
".a {
    display: flex
}
.b {
    margin-left: var(--spacing-base)
}
.c {
    flex-direction: column
}
.d {
    flex: 1
}
.e {
    width: 100%
}
.f {
    margin-bottom: 0
}
.g {
    margin-left: 0
}
.h {
    margin-right: 0
}
.i {
    margin-top: var(--spacing-base)
}
.j {
    align-items: center
}
.k {
    margin-bottom: var(--spacing-base)
}
.l {
    width: 48px
}
.m {
    height: 48px
}
.n {
    border: 1px solid var(--gray-1)
}
.o {
    border-radius: 50%
}
.p {
    margin-right: 10px
}
.q {
    overflow: hidden
}

"
`;
